bin/pscss

```diff
@@ -180,6 +180,8 @@ if ($dumpTree) {
 
     print_r(json_decode(json_encode($parser->parse($data)), true));
 
+    fwrite(STDERR, 'Warning: the --dump-tree option is deprecated. Use proper debugging tools instead.');
+
     exit();
```

Comment:
should I change that to `exit(1)` to report it as an error to use it ?

Reply:
@robocoder @Cerdic what do you think about this ?

Reply:
thinking about that again, changing the exit code does not make sense. It is only a warning, not an error.